src/theme/global.js

```diff
@@ -8,5 +8,10 @@ export const styles = {
       fontSize: { base: 'md', lg: 'lg' },
       lineHeight: 1.5,
     },
+    iframe: {
+      border: 'none',
+      maxWidth: '100%',
+      maxHeight: '450px',
```

Comment:
```suggestion
      width: '100%',
      height: '450px',
```

I think this will fix the issue